api/GPUQueue.json

```diff
@@ -344,6 +344,47 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "unique_command_buffers": {
```

Comment:
In https://github.com/mdn/browser-compat-data/pull/24811 we used the `validates_x` pattern for the feature key, so how about:

```suggestion
        "validates_command_buffer_uniqueness": {
```

Reply:
Ah, good call. Committing.